jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TasksListViewTest.java

```diff
@@ -57,7 +59,7 @@
 import static org.jbpm.workbench.ht.model.TaskDataSetConstants.*;
 
 @RunWith(GwtMockitoTestRunner.class)
-public class DataSetTasksListGridViewTest {
+public class TasksListViewTest {
```

Comment:
`TaskListViewImplTest` drop the 's' from Tasks and add Impl to match target class